contract/Makefile

```diff
@@ -163,12 +50,9 @@ fund: fund-provision-pool check-balance
 fund-osmo:
 	kubectl exec -i osmosislocal-genesis-0 -c validator -- osmosisd tx bank send faucet ${CLIENT_OSMO_ADDR} 9870000000uosmo --fees 1000uosmo -y;
```

Comment:
I suggest factoring out `EXEC_AGD` and `EXEC_OSMO` like...

https://github.com/Agoric/dapp-orchestration-basics/blob/c4ea22446ba8f1b7b62a7746d4c3c79d16fb6037/e2e-testing/Makefile#L10

Then the k8s stuff doesn't get in the way when reading stuff like...

https://github.com/Agoric/dapp-orchestration-basics/blob/c4ea22446ba8f1b7b62a7746d4c3c79d16fb6037/e2e-testing/Makefile#L92-L93

Reply:
Done.